x-pack/build_chromium/chromium_version.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { REPO_ROOT, ToolingLog } from '@kbn/dev-utils';
+import chalk from 'chalk';
+import cheerio from 'cheerio';
+import dedent from 'dedent';
+import fs from 'fs';
+import fetch from 'node-fetch';
+import path from 'path';
+import { PackageJson } from 'type-fest';
+
+type PuppeteerRelease = string;
+type ChromiumVersion = string;
+type ChromiumRevision = string;
+type ChromiumCommit = string;
+
+const log = new ToolingLog({
+  level: 'info',
+  writeTo: process.stdout,
+});
+
+// We forked the Puppeteer node module for Kibana,
```

Comment:
I actually think we could go back to using the upstream package?

Reply:
I think this was the original issue that caused us to fork it: https://github.com/puppeteer/puppeteer/issues/6634

That issue has been fixed, but it looks like it is only ported to Puppeteer 7.x :(

This tells me that in the next bump of Puppeteer, we should jump to 7.0.0 (Chromium r848005) and skipping 5.5 and 6.0